README.md

```diff
@@ -109,38 +92,31 @@ Once in the container, you can install Chiltepin in editable mode, and run the t
 cd chiltepin
 pip install -e .
 cd tests
-pytest --assert=plain --config=config.yaml --platform=chiltepin
+PYTHONPATH=.. pytest --assert=plain --config=config.yaml --platform=chiltepin
 ```
 
 NOTE: Depending on how many cores your machine has and how many you've allocated to Docker,
 you may need to modify the `cores per node` setting in the configuration yaml file to match
 your machine's specifications to get all tests to pass.
 
-# Running Parsl apps
-
-If running Chiltepin in the container, the Chiltepin spack environment is activated
-automatically when logging in to the front-end node.  If running on an HPC, it must be
-activated  manually to run Parsl Chiltepin applications
-
-```
-cd install
-. spack/share/spack/setup-env.sh
-spack env activate chiltepin
-```
 
 # Running the test suite
 The test suite is run with `pytest` and requires an editable installation of the Chiltepin
-repository.  Before running the test suite:
+repository.  From the repository root, install Chiltepin in editable mode:
 
 ```
 cd <repository root>
+```
+then (don't forget the dot at the end)
+```
 pip install -e .
 ```
```

Comment:
Since this is now included in each of the specific sections above, do we want to remove this part here? And add the comment to the specific sections above?

Reply:
Agreed.  Take another look.

Reply:
Just circling back on this comment.

Reply:
I'm not sure what you mean.  I removed that.  We still need the one on line 94 because you have to run the `pip install -e .` in the container using the pip that is in the container.  The install steps at the top don't cover that.

Reply:
I had an older version of the file open, which was out-dated. This looks good in the new version.